### DISCUSSION TITLE
ba3e4290-10e5-447f-a78c-c9825623a015

### DISCUSSION THREAD

**Discussion by giscus** [SSWConsulting/SSW.Rules.Content#9535] [Rules Discussions]
# ba3e4290-10e5-447f-a78c-c9825623a015

Branded plastic bags for giveaways and employee use promote free marketing on public transport, providing a win-win for both employees and your company.

https://www.ssw.com.au/rules/plastic-bags-branding/

**Comment by tiagov8** [comment]
We should replace those with more sustainable bags. https://www.businessinsider.com/guides/style/eco-friendly-sustainable-plastic-bag-alternatives
